sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseEditActivity.java

```diff
@@ -352,7 +352,10 @@ private void linkEventToCase() {
 				eventParticipantToSave.setResultingCaseUuid(caze.getUuid());
 				EventParticipantSaver eventParticipantSaver = new EventParticipantSaver(this);
 
-				if (!isEventLinkedToCase(caze, event)) {
+				boolean eventParticipantAlreadyExists =
```

Comment:
Unnecessary variable declaration because it's only used once; please directly call the method in the if clause

Reply:
yes, I've used it only to debug easier, I'll delete the declaration